src/components/Navbar.tsx

```diff
@@ -1,24 +1,32 @@
-export const Navbar = () => {
-  return (
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">Home</a>
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
 
-          <a
-            aria-current="page"
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
+const links = ['Home', 'People'];
```

Comment:
```suggestion
const LINKS = ['Home', 'People'];
```
we usually name constants in uppercase